scripts/check-env.sh

```diff
@@ -43,4 +67,17 @@ fi
 [ "$(go env CGO_ENABLED)" = '1' ] \
 	|| fail "CGO_ENABLED should be set to '1'"
 
+if [ -n "${YUM_CMD}" ]; then
+    check_rpm_pkg libcephfs-devel librbd-devel librados-devel
+elif [ -n "${APT_GET_CMD}" ]; then
+    check_deb_pkg libcephfs-dev librbd-dev librados-dev
+else
+    fail "error can't verify Ceph development headers"
```

Comment:
You could also consider building a simple program like this:

```go
package main

/*
#include <rados/librados.h>
*/
import "C"

func main() {
    _ = C.LIBRADOS_VERSION_CODE
}
```

This only runs  (with `go run <filename>`) when librados.h is available. Use a wrong filename in the `#include` statement to test.

Reply:
Thanks. I'll make the changes.